### PR TITLE
Removed purge flag

### LINKED ISSUE #18: Remove purge flag from instance resource in terraform
It doesn't make sense in the context of terraform.. Executing terraform destroy will cause a failure on the tier destruction.

### PR DESCRIPTION
**Pull request by franzpgarcia**
fixes #18

### CODE REVIEW

**Comment by franzpgarcia** [comment]
Only merge for v6.0.0

File: cloudca/resource_cloudca_instance.go [hunk 1]
```diff
@@ -95,13 +95,6 @@ func resourceCloudcaInstance() *schema.Resource {
 				Computed:    true,
 				Description: "The instance's memory in MB. If the compute offering is custom, this value is required",
 			},
-
-			"purge": &schema.Schema{
-				Type:        schema.TypeBool,
```
Comment (simongodard) [inline cloudca/resource_cloudca_instance.go]: I thought we would use the **Removed** attribute instead of just removing it.

Comment (simongodard) [inline cloudca/resource_cloudca_instance.go]: > // When Removed is set, this attribute has been removed from the schema
    //
    // Removed attributes can be left in the Schema to generate informative error
    // messages for the user when they show up in resource configurations.
    // This string is the message shown to the user with instructions on
    // what do to about the removed attribute.
    Removed string

**Comment by franzpgarcia** [comment]
Will do

**Comment by franzpgarcia** [comment]
Done

**Pull request merged**